README.md

```diff
@@ -6,6 +6,9 @@
 [![devDependency Status](https://david-dm.org/thelounge/lounge/dev-status.svg)](https://david-dm.org/thelounge/lounge?type=dev)
 
 # The Lounge
+<p align="center">
+	<img src="https://raw.githubusercontent.com/thelounge/thelounge.github.io/master/img/screenshots/example_css.png" width="550" />
```

Comment:
Why not directly https://thelounge.github.io/img/screenshots/example_css.png?
Also, `<nitpick>`there is no trailing slash on `img` elements in HTML5 :D`</nitpick>`.

Finally, add a new line between `# The Lounge` and the `p` element, some Markdown parsers (like npm's one at one point, not sure if still true) don't like that too much.